dev-python/docker-py/docker-py-2.0.1.ebuild

```diff
@@ -38,19 +25,25 @@ RDEPEND="
 	$(python_gen_cond_dep '>=dev-python/backports-ssl-match-hostname-3.5[${PYTHON_USEDEP}]' 'python2_7' 'python3_4' )
 	$(python_gen_cond_dep '>=dev-python/ipaddress-1.0.16[${PYTHON_USEDEP}]' 'python2_7' )
 "
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	test? (
+		${RDEPEND}
+		>=dev-python/mock-1.0.1[${PYTHON_USEDEP}]
+		dev-python/pytest-runner[${PYTHON_USEDEP}]
+		>=dev-python/pytest-2.9.1[${PYTHON_USEDEP}]
+	)
+	doc? ( dev-python/recommonmark[${PYTHON_USEDEP}]
+		>=dev-python/sphinx-1.4.6[${PYTHON_USEDEP}] )
```

Comment:
can you do the parentheses style as in `test?` before? i.e. opening and closing parentheses are by themselves